GruntFile.js

```diff
@@ -251,26 +241,6 @@ module.exports = function (grunt) {
                 }
             }
         },
-        shell: {
-            options: {
-                stdout: true
-            },
-            bootstrapMapServices: {
-                command: 'python scripts/publish_mxd.py'
```

Comment:
**question**: this still seems useful. I'm curious about your decision to remove it.

Reply:
I couldn't find the `publish_mxd.py` file anywhere in the project. I assumed that this was just clean up of unused code.

Reply:
It looks like it was deleted in this pr. https://github.com/agrc/Crash-web/blob/77abcd559638a41fe6b050823ebec17028f21ac5/scripts/publish_mxd.py

Reply:
Do you want me to bring it back? Looks like py2.

Reply:
😳 i think so to have something to update to py3